terraform/projects/app-ecs-albs/main.tf

```diff
@@ -129,19 +129,127 @@ resource "aws_lb_listener" "monitoring_external_listener" {
   }
 }
 
+resource "aws_lb" "alertmanager_external_alb" {
+  name               = "${var.stack_name}-alertmanager"
+  internal           = true
+  load_balancer_type = "application"
+  security_groups    = ["${data.terraform_remote_state.infra_security_groups.aletmanager_external_sg_id}"]
+
+  subnets = [
+    "${element(data.terraform_remote_state.infra_networking.public_subnets, 0)}",
+    "${element(data.terraform_remote_state.infra_networking.public_subnets, 1)}",
+    "${element(data.terraform_remote_state.infra_networking.public_subnets, 2)}",
+  ]
+
+  tags = "${merge(
+    local.default_tags,
+    var.additional_tags,
+    map("Stackname", "${var.stack_name}"),
+    map("Name", "${var.stack_name}-ecs-monitoring")
+  )}"
+}
+
+resource "aws_lb_listener" "alertmanager_listener" {
+  load_balancer_arn = "${aws_lb.alertmanager_external_alb.arn}"
+  port              = "80"
+  protocol          = "HTTP"
+
+  default_action {
+    target_group_arn = "${aws_lb_target_group.alertmanager_endpoint.arn}"
+    type             = "forward"
+  }
+}
+
+resource "aws_lb_target_group" "alertmanager_endpoint" {
+  name     = "${var.stack_name}-alertmanager"
+  port     = "9093"
+  protocol = "HTTP"
+  vpc_id   = "${data.terraform_remote_state.infra_networking.vpc_id}"
+
+  health_check {
+    interval            = "10"
+    path                = "/"
+    matcher             = "200"
+    protocol            = "HTTP"
+    healthy_threshold   = 2
+    unhealthy_threshold = 2
+    timeout             = "5"
+  }
+}
+
+
+resource "aws_lb_listener" "paas_proxy_listener" {
+  load_balancer_arn = "${aws_lb.alertmanager_external_alb.arn}"
+  port              = "8080"
+  protocol          = "HTTP"
+
+  default_action {
+    target_group_arn = "${aws_lb_target_group.paas_proxy_endpoint.arn}"
+    type             = "forward"
+  }
+}
+
+resource "aws_lb_target_group" "paas_proxy_endpoint" {
+  name     = "${var.stack_name}-paas-proxy"
+  protocol = "HTTP"
+  port = "8080"
+  vpc_id   = "${data.terraform_remote_state.infra_networking.vpc_id}"
+
+  health_check {
+    interval            = "10"
+    path                = "/health"
+    matcher             = "200"
+    protocol            = "HTTP"
+    healthy_threshold   = 2
+    unhealthy_threshold = 2
+    timeout             = "5"
+  }
+}
+
 ## Outputs
 
 output "monitoring_external_tg" {
   value       = "${aws_lb_target_group.monitoring_external_tg.arn}"
   description = "External Monitoring ALB target group"
 }
 
-output "dns_name" {
+output "alertmanager_external_tg" {
+  value       = "${aws_lb_target_group.alertmanager_endpoint.arn}"
+  description = "External Alertmanager ALB target group"
+}
+
+output "prometheus_alb_zoneid" {
+  value = "${aws_lb.monitoring_external_alb.zone_id}"
+  description = "External Alertmanager ALB target group"
```

Comment:
This description is inaccurate. Copypaste error?

Reply:
Yep... I hate typing when I can copy & reword :( 

I wanted to make it work, will clean these up to ensure the description matches the output 👍